TROUBLESHOOTING.md

```diff
@@ -101,16 +101,32 @@ bind 'set horizontal-scroll-mode off'
 Terminal multiplexer tools like GNU `screen` or `tmux` are sometimes reported to fix the issues, though at other times they are reported as the _cause_ of the problem. They have their own configuration files to take into account.
 
 Further reference:
-* https://stackoverflow.com/questions/1133031/shell-prompt-line-wrapping-issue
-* https://superuser.com/questions/46948/any-way-to-fix-screens-mishandling-of-line-wrap-maybe-only-terminal-app
-* https://unix.stackexchange.com/questions/105958/terminal-prompt-not-wrapping-correctly
-* https://unix.stackexchange.com/questions/529377/terminal-long-line-wrapping
-* https://github.com/microsoft/WSL/issues/1436
+
+- https://stackoverflow.com/questions/1133031/shell-prompt-line-wrapping-issue
+- https://superuser.com/questions/46948/any-way-to-fix-screens-mishandling-of-line-wrap-maybe-only-terminal-app
+- https://unix.stackexchange.com/questions/105958/terminal-prompt-not-wrapping-correctly
+- https://unix.stackexchange.com/questions/529377/terminal-long-line-wrapping
+- https://github.com/microsoft/WSL/issues/1436
 
 If nothing seems to help, consider also using a different client-side terminal application:
-* Linux: xterm, KDE Konsole, GNOME Terminal
-* Mac: Terminal, iTerm2
-* Windows: PowerShell, PuTTY, WSL (Windows Subsystem for Linux)
+
+- Linux: xterm, KDE Konsole, GNOME Terminal
+- Mac: Terminal, iTerm2
+- Windows: PowerShell, PuTTY, WSL (Windows Subsystem for Linux)
+
+## "balena ssh" ends with permission denied when using RSA keys
+
+RSA keys generated with this command from github documentation: `ssh-keygen -t rsa -b 4096 -C "your_email@example.com"`
+
+It failed with this error:
+`Offering public key: /Users/<username>/.ssh/id_rsa RSA`
+`send_pubkey_test: no mutual signature algorithm`
+
+Possible Solutions
+
+- Either add PubkeyAcceptedKeyTypes +ssh-rsa to SSH configuration or use a different algorithm for the key (ecdsa or ed25519 should work fine)
+
+- Change `.ssh/config` to refer to the custom named identify file? e.g. id_balena / id_balena.pub
```

Comment:
* Did you mean a comma or a colon instead of a question mark? I.e. `file?` → `file,`
* Typo: `identify file` → `identity file`
* How does a custom identity file help? I guess that the custom identity file could not be RSA, and would have to be ecdsa or ed25519, just like the primary / default identity file, right? If so, we may want to clarify that a custom identity file may be helpful if the user wishes to keep their their primary / default RSA identity file unchanged, and create an alternative file with a different key type for use with balena.
  * I also think we would need to provide an example of how the `Host` section should look like in the `.ssh/config` file. What value should the user use for `Host`? Does it work with `Host *.balena`? I assume that the custom identity file configuration would have to fall under some such `Host` configuration. Otherwise, it would effectively replace the primary / default identity file for all hosts, not just balena.